lib/sanitized.js

```diff
@@ -0,0 +1,14 @@
+function sanitized(array){
```

Comment:
`sanitize` should take a string as input and sanitized the string. You'd call it before creating the array of characters.